### PR TITLE
Improve readability of Administrators page

### PR DESCRIPTION
**Pull request by tomwarwick**
This commit is intended to improve the readability of the Administrators page.

- Better define the role of an Integrator
- Remove references to the "old ways" of URL handling in previous versions of TYPO3

### CODE REVIEW

File: Documentation/NextSteps/Administrators/Index.rst [hunk 1]
```diff
@@ -8,14 +8,16 @@
 For Integrators
 ===============
 
+The role of a TYPO3 Integrator is that of someone who's responsibilities
```
Comment (sypets) [inline Documentation/NextSteps/Administrators/Index.rst]: I slightly disagree with role of "integrator". Perhaps we can put that on hold for now. I will continue with the other PRs. Actually, to be on the safe side, it would be easiest to just point to the certification page to define integrator, as I did before: https://typo3.org/certification/integrator/

---

Please also see what certification defines:

> The TYPO3 CMS Certified Integrator develops the template for a website, configures all the necessary extensions and creates the access rights for backend users. The TYPO3 CMS Certified **Integrator does not install TYPO3 on the web server** and does not program extensions. The TYPO3 CMS Certified Integrator knows how to configure the pre-installed TYPO3 using the installer.

https://typo3.org/certification/integrator/

You wrote:

> The role of a TYPO3 Integrator is that of someone who's responsibilities 
include the **installation**, configuration and ongoing **maintenance** of a TYPO3
based website.

To summarize, I see role of integrator as someone:

* who integrates a HTML Template into TYPO3 (using Fluid / TypoScript etc.) - as the name "integrator" says, as is described for example in ["Sitepackage Tutorial"](https://docs.typo3.org/typo3cms/SitePackageTutorial/)
* configures TYPO3 (including extensions)

As the certification page explicitly says, installation is not job of integrator. I would see that as role of "system administrator" (for server related tasks such as setting up webserver etc.) and / or "DevOps" as using Composer / deployment tools to "deploy" TYPO3.

Please also see related issue:  https://github.com/TYPO3-Documentation/T3DocTeam/issues/71 and the new ["Tutorials & Guides"](https://docs.typo3.org/typo3cms/GuidesAndTutorials/Index.html) page

Comment (tomwarwick) [reply Documentation/NextSteps/Administrators/Index.rst]: I guess the problem we face is the fact the page itself defines the role as an Administrator, yet the certification defines it as an Integrator.

Recommend we disregard/delet this PR.

**Comment by sypets** [comment]
Ok, I will close. We can resume at any time.
